Lib/test/libregrtest/utils.py

```diff
@@ -124,15 +124,6 @@ def clear_caches():
         if stream is not None:
             stream.flush()
 
-    # Clear assorted module caches.
-    # Don't worry about resetting the cache if the module is not loaded
```

Comment:
This comment isn't specific to distutils,. But it's probably not worth putting it back – it's not that hard to see what the code below does.